pkg/operator/defaults.go

```diff
@@ -14,7 +14,7 @@ var (
 		"v0.18.3",
 		"v0.18.4",
 		"v0.19.0",
-		"v0.20.0",
+		"v0.21.0",
```

Comment:
This should add a new version instead of changing the old one

Reply:
woops, did some find / sed here and missed excluded this one.